addon/helpers/array.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export function array(params=[]) {
+  return params;
```

Comment:
If you wrap this with `Ember.A()`, does it still appear truthy in the template when empty?